src/main/java/com/elasticbox/jenkins/triggers/github/PullRequestBuildListener.java

```diff
@@ -145,4 +146,12 @@ public void onCompleted(AbstractBuild<?, ?> build, TaskListener listener) {
         postStatus(build, pullRequest, status, message);
     }
 
+    private boolean isRemovableAction(Action action, String sha) {
```

Comment:
This method did not test for the removable condition. This method tests if an action is the one described in the sha. Maybe you could rename it accordingly?